backend/inspirehep/migrator/tasks.py

```diff
@@ -275,6 +279,11 @@ def process_references_in_records(uuids):
     return uuids
 
 
+@shared_task(ignore_results=False, queue="migrator", acks_late=True)
+def process_conferences_papers(uuids):
+    return process_record_linked_to_conferences(uuids)
```

Comment:
I think it's better to generalize the `recalculate_citations` into `recalculate_records_relations` in which you do both.